app/presenters/formats/service_sign_in_presenter.rb

```diff
@@ -0,0 +1,133 @@
+module Formats
+  class ServiceSignInPresenter
+    attr_reader :content
+
+    def initialize(content)
+      @content = content.deep_symbolize_keys
+    end
+
+    def render_for_publishing_api
+      {
+        schema_name: "service_sign_in",
+        rendering_app: "government-frontend",
+        publishing_app: "publisher",
+        document_type: "service_sign_in",
+        locale: locale,
+        update_type: update_type,
+        change_note: change_note,
+        base_path: base_path,
+        routes: routes,
+        title: title,
+        description: description,
+        public_updated_at: public_updated_at,
+        details: details,
+      }
+    end
+
+    def content_id
+      begin
+        content_item["content_id"]
```

Comment:
I would hard code the `content_id` to avoid having to talk to content-store/publishing-api.

Reply:
To be fair, if we were to hard-code the content-id, someone would still have to do a manual lookup of the content-id. We're trying to save them the trouble. Also, we're trying to avoid copy and paste errors as much as possible.

Happy to through our reasoning and approach with you in person.

Reply:
> someone would still have to do a manual lookup of the content-id

When would you need to do that?

Reply:
When you republish the content-item. 
Are you suggesting that a random uuid is generated before the first publish, and then added to the YAML config?